pertpy/tools/_augur.py

```diff
@@ -877,6 +877,9 @@ def predict_differential_prioritization(
             & set(permuted_results1["summary_metrics"].columns)
             & set(permuted_results2["summary_metrics"].columns)
         )
+
+        cell_types = list(cell_types)
```

Comment:
why this?

Reply:
This fixes the initial bug: `TypeError: Passing a set as an indexer is not supported. Use a list instead.`